.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
             echo "MAIL_PORT=25" >> ./.env
             echo "MAIL_SENDER=change@me.com" >> ./.env
             echo "NODE_ENV=development" >> ./.env
-            echo "REACT_APP_GRAPHQL_API_URL=http://0.0.0.0:5433/graphql" >> ./.env
+            echo "REACT_APP_GRAPHQL_API_URL=http://0.0.0.0:5434/mobydq/api/v1/graphql" >> ./.env
```

Comment:
Hi Thomas,
What about we rename this env variable `REACT_APP_GRAPHQL_API_URL` into `FLASK_API_URL`? I think it would look more consistent with the other env variables such as `DATABASE_URL` and `GRAPHQL_URL`.

Reply:
Yeah, that would make sense. I've updated it in 3cbf8c7dfe6bbaf4220b246c049bb8330a67f447.